ush/config.sh.RTMA_CONUS_3km

```diff
@@ -147,7 +147,7 @@ HYBENSMEM_NMIN=66
 HALO_BLEND=20
 IO_LAYOUT_Y=4
 PPN_RUN_REF2TTEN=1
-PPN_RUN_NONVARCLDANL=1
+PPN_RUN_NONVARCLDANL=${IO_LAYOUT_Y}
```

Comment:
Hi Guoqing and Ming,

I noticed that you already approved and merged this PR 4 mins ago. 
I just would like to confirm with you that: this variable "IO_LAYOUT_Y" had been already added and defined with default values in the running scripts and workflow files, correct?  Or the user needs to add it to the scripts by themselves when running their own experiments?
Thank you!

- Gang

Reply:
@GangZhao-NOAA IO_LAYOUT_Y is defined in config.sh.RTMA_CONUS_3km. 
Both GSL and EMC use the same IO_LAYOUT_Y value (4 for CONUS, 10 for NA_3km) and users don't need to test/make any changes on this. Thanks!

Reply:
I see. Thank you, Guoqing for the explanation!

-Gang